nornir/core/task.py

```diff
@@ -174,6 +175,7 @@ def __init__(
         self.exception = exception
         self.name = None
         self.severity_level = severity_level
+        self.timestamp = datetime.datetime.now(datetime.timezone.utc)
```

Comment:
I am curious about this, you added it but don't seem to be using it anywhere

Reply:
The primary use-case for me is "poor man's telemetry", meaning running getters on a schedule, timestamping them, and using a pipeline to eventually process them into other datastores (ELK, etc).  Having the timestamp of when the result was generated helps to order the results, in the case where a large number of them result in batching inside the producer.

If you think that's unnecessary, I'd be happy to insert the timestamp inside _`send_single_result`

Reply:
It's all good, just realized you added this to the `Result` class, I misread it yesterday and thought it was being added to the `Task` class.